roman-numerals/roman_numerals.c

```diff
@@ -1 +1,43 @@
 #include "roman_numerals.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct{
+    unsigned int base;
```

Comment:
this is fine - but just as note for gea we usually stick with the stdint types - for this one the equivalent would be uint16_t in gcc